gc/base/Configuration.cpp

```diff
@@ -169,9 +169,11 @@ MM_Configuration::initializeEnvironment(MM_EnvironmentBase* env)
 	case gc_modron_allocation_type_tlh:
```

Comment:
Sure, but then we have the awkward problem that there are no valid allocation types. I'm not really familiar with this code, but maybe we should introduce a generic allocation type, and initialize with an `MM_AllocationInterfaceGeneric` here.

As well, It looks like the entirety of `omrgcconsts.h` is in need of ifdefs.